examples/lib/stories/components/composability.dart

```diff
@@ -41,20 +41,20 @@ class Composability extends BaseGame with TapDetector {
 
   @override
   Future<void> onLoad() async {
-    _parent = ParentSquare(Vector2.all(200), Vector2.all(300))
+    _parent = ParentSquare(Vector2.all(400), Vector2.all(250))
```

Comment:
What is the reason to change things in this class?

Reply:
It was close to the edge of the screen, so part of the picture was cut off when it rotated. And I wanted to be able to see  it nice and clear, because this example is the only real visual test for nest positioned components, and for the debug paint mode.